Formula/swiftformat.rb

```diff
@@ -0,0 +1,14 @@
+class Swiftformat < Formula
+  desc "Formatting tool for Swift source code"
+  homepage "https://github.com/nicklockwood/SwiftFormat"
+  url "https://github.com/nicklockwood/SwiftFormat/archive/0.17.tar.gz"
+  sha256 "68cfc0f783ca331a1fda1c8b5c5a670b6c42be72a44a34b519d61d308b2fe8ea"
+
+  def install
+    bin.install "CommandLineTool/swiftformat"
+  end
+
+  test do
+    system "#{bin}/swiftformat"
```

Comment:
Please write a test that does something more substantial than e.g. printing the help. You can write a test Swift file and call `swiftformat` on it.